Packages/MIES/MIES_ExperimentDocumentation.ipf

```diff
@@ -489,7 +489,7 @@ static Function [WAVE colIndizes, variable rowIndex] ED_FindIndizesAndRedimensio
 	numKeyCols = DimSize(key, COLS)
 	lastValidIncomingKeyRow = DimSize(incomingKey, ROWS) - 1
 
-	Make/FREE/U/I/N=(DimSize(incomingKey, COLS)) indizes = NaN
+	Make/FREE/D/N=(DimSize(incomingKey, COLS)) indizes = NaN
```

Comment:
@t-b So IP9 converts NaN in unsigned integer wave type, not all wave types, is that right?

Reply:
You can not store a NaN in an integer wave (you can only store it in floating point waves), so there has to be some conversion going on. And as the code is as it was difficult to grasp, I've decided to use a double wave instead which can store NaN.